.eslintrc.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+
+  root: true,
+
+  parserOptions: {
+    'ecmaVersion': 2017,
+  },
+
+  extends: [
+    '@metamask/eslint-config',
+    '@metamask/eslint-config/config/nodejs',
+    '@metamask/eslint-config/config/mocha',
```

Comment:
I think this requires `eslint-plugin-mocha`? That package doesn't seem to have been added.

Reply:
Oh wait, it was already there 🤦‍♂